sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/export/BatchSpanProcessor.java

```diff
@@ -277,10 +272,12 @@ private void exportBatches(ArrayList<ReadableSpan> spanList) {
 
     // Exports the list of SpanData to the SpanExporter.
     @SuppressWarnings("BooleanParameter")
-    private void onBatchExport(final List<SpanData> spans) {
+    private int onBatchExport(final List<ReadableSpan> spans) {
```

Comment:
I'm thinking of renaming `onBatchExport` to `exportBatch` if we proceed with these changes